test/test.externalclient.ts

```diff
@@ -77,6 +77,32 @@ describe('ExternalAccountClient', () => {
       forceRefreshOnFailure: true,
     };
 
+    const invalidWorkforceIdentityPoolClientAudiences = [
+      '//iam.googleapis.com/projects/123/locations/global/workloadIdentityPools/pool/providers/oidc',
+      '//iam.googleapis.com/projects/123/locations/global/workforcepools//providers/oidc',
```

Comment:
We want to change one thing, otherwise we don't know if it's failing for the same reason as a previous audience (no pool name provided) or not. 

//iam.googleapis.com/projects/123/locations/global/workforce**P**ools//providers/oidc